benches/solidity.rs

```diff
@@ -135,7 +135,41 @@ fn baseline(c: &mut Criterion) {
     group.finish()
 }
 
-criterion_group!(computation, baseline, odd_product, triangle_number);
+fn fibonacci(c: &mut Criterion) {
+    let mut group = c.benchmark_group("fibonacci_iterative");
+    group.sample_size(20);
+
+    for n in [128u32, 192, 256, 320] {
+        let args_scale = [(n, format!("{n}"))];
+        bench_solang(&mut group, "FibonacciIterative", "fib", &args_scale);
+
+        let args_evm = [(vec![DynSolValue::Uint(U256::from(n), 32)], format!("{n}"))];
+        bench_evm(&mut group, "FibonacciIterative", "fib", &args_evm);
```

Comment:
Do `evm` before `solang` so that it appears as the baseline in the resulting benchmark table

Reply:
Good point lets keep this consistent